ui/packages/catalog-ui-search/src/main/webapp/react-component/dropdown/dropdown.js

```diff
@@ -57,18 +60,30 @@ const getPosition = (viewport, rect) => {
   return { width, left: x, ...pos }
 }
 
+const foreground = props => {
+  if (props.theme.backgroundDropdown !== undefined) {
+    return readableColor(props.theme.backgroundDropdown)
```

Comment:
We have to do this in every component now?

Reply:
I think we should precompute this as part of the theme but that is a very common API so I wanted to coordinate with others before I added anything. I mainly didn't want people depending on these as helpers which is why they aren't in a common location.